openhands/core/main.py

```diff
@@ -153,6 +158,34 @@ async def run_controller(
         f'{agent.llm.config.model}, with task: "{task_str}"'
     )
 
+    # Copy files from workspace_base to workspace_mount_in_sandbox
+    if config.workspace_base and hasattr(runtime, 'copy_to'):
```

Comment:
How do you mean? Should we do these operations on `__init__` and `close`?

Reply:
Actually we should just remove this.

Our current remote runtime implementation just doesn't do any disk mounting--the user is expected to run e.g. `git clone` to get a workspace. We're actually modifying the core UI to better support pulling in git repos and uploading zip files, since remote runtimes are becoming more important.

Reply:
I'm happy to revert this - the challenge though is that none of the integration tests will pass without it. This is the same problem as the `remote` runtime, though, so I guess we can wait for integration tests to work with remote runtimes.

Reply:
Huh--our integration tests do work with our remote runtime, and I don't think we do any copying...

@xingyaoww do you know how this works?

Reply:
I'm surprised by this. The integration tests look to me like they require the sandbox to be executing on the same host as the test harness. I noticed this in the required bind mount on `workspace_base` and the HTTP server access in `test_browse_internet`.